drink/src/lib.rs

```diff
@@ -4,54 +4,33 @@
 #![warn(missing_docs)]
 
 pub mod errors;
-pub mod runtime;
-pub mod sandbox;
-pub use sandbox::*;
+pub mod pallet_contracts_debugging;
 #[cfg(feature = "session")]
 pub mod session;
 
 #[cfg(feature = "macros")]
 pub use drink_test_macro::{contract_bundle_provider, test};
 pub use errors::Error;
-use frame_support::traits::fungible::Inspect;
-pub use frame_support::{
-    sp_runtime::{AccountId32, DispatchError},
-    weights::Weight,
+pub use frame_support;
+pub use ink_sandbox::{
```

Comment:
just `ink_sandbox::*`?

Reply:
yeah I just thought that `api as sandbox_api` would make it a bit clearer